main/pipeline/pipeline.cc

```diff
@@ -1264,4 +1264,44 @@ vector<ast::ParsedFile> autogenWriteCacheFile(const core::GlobalState &gs, const
 #endif
 }
 
+void printUntypedBlames(const core::GlobalState &gs, const UnorderedMap<long, long> &untypedBlames,
+                        const options::Options &opts) {
+#ifndef SORBET_REALMAIN_MIN
+
+    rapidjson::StringBuffer result;
+    rapidjson::Writer<rapidjson::StringBuffer> writer(result);
+
+    writer.StartArray();
+
+    for (auto &[symId, count] : untypedBlames) {
+        auto sym = core::SymbolRef::fromRaw(symId);
+
+        writer.StartObject();
+
+        writer.String("path");
+        if (sym.exists() && sym.loc(gs).exists()) {
+            writer.String(std::string(sym.loc(gs).file().data(gs).path()));
+        } else {
+            writer.String("<none>");
+        }
+
+        writer.String("owner");
+        auto owner = sym.owner(gs).show(gs);
+        writer.String(owner);
```

Comment:
It's _possible_ we want to tweak this output.

I think that for a method like

```ruby
A::B::C.foo
```

This will show up as

```json
{
    "owner": T.class_of(A::B::C),
    "name: "foo"
}
```

That makes it a _little_ tricky to work with this data (e.g., you can't group by owner unless you process the data a bit.

Some possibilities:

- We add another field like `ownerAttached` and populate it by following the chain of `attachedClass` all the way to the bottom (e.g., `T.class_of(T.class_of(T.class_of(A)))` gets unwrapped all the way to `A`)
- We make `owner` be like `ownerAttached`, and make `name` indicate whether it's a singleton or instance method somehow(?)
- We make `owner` be like `ownerAttached` and add another field like `exactOwner` or something
- We make `owner` be like `ownerAttached` and keep `name` as is, but a add another field like `absoluteName` or something that has the name like it appears in the old `suggestSig` code
- We give up on all of this and just only aggregate by file/package, not by owner. Realistically, aggregating by file is _maybe_ enough? Except that a single class will have some methods from an RBI and some from ruby source files

Reply:
For simplicity done the following currently:
1. Added package information
2. Leaving `owner` as-is for simplicity. Prefer post-processing. If it ends up being a hurdle, we will come back to this and tidy it up.